### PR TITLE
Features/1703 rec project crash

### PR DESCRIPTION
**Pull request by tomkamin**

### CODE REVIEW

File: Database/ReleaseScript/0259 - Add funding type data for reclamation.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+insert into dbo.FundingTypeData(TenantID, FundingTypeID, FundingTypeDisplayName, FundingTypeShortName, SortOrder)
```
Comment (bGrass) [inline Database/ReleaseScript/0259 - Add funding type data for reclamation.sql]: This table is going to be deleted on Develop as soon as I get my active branch merged.  Depending on timing, this file may need to be deleted prior to merge.

**Comment by bGrass** [comment]
This feels like a lot of code for something that should only be encountered by Sitka devs at the very beginning of a project.  Project Firma has a convention of using SetErrorForDisplay() to display user-facing error messages without cluttering up the ViewData and .cshtml files.  In this case, however, I think a few Check.Assert()s would be adequate to help a developer hone in on these config issues.

Also, the same dependencies that you're trapping for exist on the ProjectUpdate workflow as well.

**Comment by tomkamin** [comment]
> 
> 
> This feels like a lot of code for something that should only be encountered by Sitka devs at the very beginning of a project. Project Firma has a convention of using SetErrorForDisplay() to display user-facing error messages without cluttering up the ViewData and .cshtml files. In this case, however, I think a few Check.Assert()s would be adequate to help a developer hone in on these config issues.
> 
> Also, the same dependencies that you're trapping for exist on the ProjectUpdate workflow as well.

The performance measures and classifications can be setup through the UI, so I assumed that was a BA/PO type task. Our BA was the one that initially discovered these issues when trying to create a project.

I am more than happy to add these checks to the ProjectUpdate workflow as well. We are just trying to get a project created. I would expect that you would have a smaller chance of encountering these issues in ProjectUpdate workflow due to needing a project to be created first.

**Comment by StewLGSitka** [comment]
This may be a bit more than we need but it definitely works, and will help someone in the future. I'm accepting this.

**Pull request merged**